.github/workflows/deploy_dbt_docs.yaml

```diff
@@ -32,14 +32,9 @@ jobs:
           aws-region: us-east-1
 
       - name: Generate docs
-        # Target is currently set to CI because we expect this action to be
-        # run against the long-lived data-catalog branch, but we should change
-        # this to prod when we merge that branch into master
-        run: dbt docs generate --target ci
+        run: dbt docs generate --target "$TARGET"
         working-directory: ${{ env.PROJECT_DIR }}
         shell: bash
-        env:
-          HEAD_REF: data-catalog
```

Comment:
**question (dumb):** This does _not_ need to be set for master right? It was a specific setup for the long-lived catalog branch?

Reply:
That's right! We only need `HEAD_REF` to be set on CI in order to properly generate the schema; dev and prod follow their own pattern that doesn't rely on this env var.

https://github.com/ccao-data/data-architecture/blob/0efef26f1718f43741694b9443a1fca973589e7e/dbt/macros/generate_schema_name.sql#L29-L34

Normally we pull `HEAD_REF` automatically from the `GITHUB_HEAD_REF` variable that is [included by default](https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables) in `pull_request` worfklow events, but we needed to set `HEAD_REF` here manually because we wanted to pretend that `data-catalog` was a CI branch for the sake of testing. (Unfortunately this didn't end up mattering, because GitHub Pages can only deploy from workflows that are on the main branch 😅 )

https://github.com/ccao-data/data-architecture/blob/0efef26f1718f43741694b9443a1fca973589e7e/.github/actions/configure_dbt_environment/action.yaml#L10-L16